### ISSUE TITLE
Wastraat

### ISSUE THREAD

**Issue by rubenvdlinde** [VNG-Realisatie/huwelijksplanner#263]
Wasstraat definitie
- 1 Koppelen aan protocomponent
- 2 opschonen code
- 3 inrichten ci/cd straat -> deploy naar docker-hub utrecht. Daarvoor moet contantijn iets inregelen.
- 4 schrijven van postman test
- 5 datamodel (doorlinken vanuit reame en redoc)

**Comment by rubenvdlinde** [comment]
@rjzondervan Hoewel het wasstraat verhaal inmiddels geweest is kunnen we dit issue nu even aanhouden voor het bijwerken van de datamodellen op de verschillende repositories

**Comment by CMasselink** [comment]
@rubenvdlinde bijwerken datamodellen zijn review/rest punten van de overige userstory's toch?

**Comment by rubenvdlinde** [comment]
Yup, en dat lopen we per component als onderdeel van de DOD bij deployen. Los vand at we daar an zich wat strakker op mogen zitten kan dit koepel issue wat mij betreft dan ook eigenlijk weg. Het oorspronkelijke doel (alle componenten bijwerken) is immers inmiddels gedaan. 

Ze beschikken daarmee dan ook over de benodigde code, er zijn losse issues om e.a. nog te configureren en aan te zetten (die heb ik gisteren geprobeerd inzichtelijk te maken) maar dat kunnen we mee nemen in de deploy die we moeten doen om alle componenten van auth te voorzien (dus vanmiddag of morgen ochtend).

Wat mij betreft zouden we dit issue dan ook kunnen sluiten en eventuele vervolg vragen los inschieten.

**Comment by matthiasoliveiro** [comment]
Volgens mij kunnen we deze afsluiten maar even een dubbel check @CMasselink
